packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -23,19 +23,22 @@
 %global gem_name smart_proxy_salt
 %global plugin_name salt
 
-%global foreman_proxy_min_version 1.25
+%global foreman_proxy_min_version 2.5
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
 %global foreman_proxy_statedir %{_root_localstatedir}/lib/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
 %global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
you can drop this line, it's unused now.

Reply:
So, [this](https://github.com/theforeman/foreman-packaging/pull/6968/files#diff-7f2b45922f25a10c95613ac59ae09ca985b02817e57bc926206b8815a454747aR59) line can be dropped too?

Reply:
No?

just the ` %global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d` line is not necesary anymore, given that you don't use dynflow_core anymore.

Reply:
Sure, sry! The highlighting of my own change irritated me. It would've wondered me if the `min_version` wasn't necessary anymore.